.npmrc

```
@@ -1,4 +1,3 @@
-node-version=20.11.1
 engine-strict=true
 ignore-scripts=true
 ignore-dep-scripts=true
```

```suggestion
ignore-dep-scripts=true
use-beta-cli=true
```

test

Refs: https://pnpm.io/9.x/npmrc#use-beta-cli